### ISSUE TITLE
RGB color sync issues and delays between my components

### ISSUE THREAD

**Issue by alasharim** [antonpup/Aurora#2040]
### Expected Behaviour
I have an Asrock z570 Extreme4(Case RGB and Fans RGB) --> lets call just it "Case RGB"  that is connected to aurora through Razer Chroma using Asrock Polychrome.

i can control my Case RGB in Aurora by changing the mouses's logo color.. oddly enough that works

so overall im controlling 4 things:
Keyboard --> Razer Blackwidow Elite (Works fine)
Headphones --> G933 Logitech (Works fine)
GPU/RAM --> Aorus GTX 2080 Super/Trident G.Skill using RGBFusion (Works fine)/(Works fine)

Headphone/Mouse/Case RGB --> G933/Razer Viper Elite/Cougar Case and fans (Works fine)/(Works fine)/**(Doesn't Work fine)**
Controlled by changing the color of the logo of any set mouse

the expected behavior would be if i chose for all colors to be red they would be red then if switch them blue they would as expected switch blue

### Actual Behaviour
the Actual behavior would be if i chose for all colors to be red they would be red then if switch them blue everything switches to blue except the case

### Frequency
always

### Aurora Version:
Beta v0.7.2
[2020-05-26 09.40.06.log](https://github.com/antonpup/Aurora/files/4680611/2020-05-26.09.40.06.log)
![IMG_20200526_104117](https://user-images.githubusercontent.com/26365097/82873970-f88c7f00-9f3d-11ea-9629-28831d8429ee.jpg)

**Comment by timami** [comment]
I know that ASRock is technically not supported yet. You could upvote this https://github.com/antonpup/Aurora/issues/1424

**Comment by alasharim** [comment]
its is supported through razer chroma along with Gskill Trident

**Comment by timami** [comment]
> its is supported through razer chroma along with Gskill Trident

Right, but I am pretty sure that doesn't mean that the support transfers to aurora. I am curious if the devs can use asrock because it's using the razer api though.
